### PR TITLE
edit readme

### PR DESCRIPTION
**Pull request by DanielGluskin**
- edit bullets 1, 2, 3 & 6. (4 & 5 are labeled as draft).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -276,13 +276,16 @@ services:
...
-//docker-compose file
-```
-//Put global setup code here
+
+```javascript
+  // jest.config.js
+  globalSetup: './example-application/test/global-setup.js'
+
+  // global-setup.js
+  await dockerCompose.upAll();
```
Comment (goldbergyoni) [inline README.md:285]: That's mystery - How come we can invoke dockerCompose liks this?

Put more context, require the relevant package, give a bit more context

File: README.md [hunk 2]
```diff
@@ -304,12 +307,14 @@ services:
 <br/>
 
 <details><summary>✏ <b>Code Examples</b></summary>
-//docker-compose file
-```
-// Put teardown code that shows how we check for CI
+
+```javascript
+  if (isCI) {
```
Comment (goldbergyoni) [inline README.md:312]: Give more context - Put comments above, "//global-teardown.js - Where we clean-up after all tests", show jest config like you did before

File: README.md [hunk 3]
```diff
@@ -399,14 +404,12 @@ services:
 <br/>
 
 <details><summary>✏ <b>Code Examples</b></summary>
-//docker-compose file
-```
-// Show global-setup conditional migration command
-//Show npm script
+
+```javascript
+  await npmCommandAsPromise(['db:migrate']);
```
Comment (goldbergyoni) [inline README.md:409]: The reader won't understand where is this coming from, don't count on reading everything before. Give more context.

Put jest config, put comment above "// In the global setup we also install all the tables"

**Review by goldbergyoni — CHANGES_REQUESTED** [review]
See comments

**Comment by DanielGluskin** [comment]
@goldbergyoni

**Comment by goldbergyoni** [comment]
@DanielGluskin Tests fail, can't merge

**Comment by DanielGluskin** [comment]
> @DanielGluskin Tests fail, can't merge

@goldbergyoni they fail with the target branch - not related to my PR. I'll try to fix that tomorrow.

**Review by goldbergyoni — APPROVED** [review]

**Pull request merged**